### ISSUE TITLE
Angular Universal - ReferenceError: document is not defined

### ISSUE THREAD

**Issue by striky1** [fullcalendar/fullcalendar#7352]
### Reduced Test Case

https://stackblitz.com/github/fullcalendar/fullcalendar-examples/tree/main/angular15?file=README.md

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

Angular

### Bug Description

👋🏻 Hola!

during the building of SSR (Server Side Rendering) via Angular Universal, I was getting next error:

`ReferenceError: document is not defined` in `vendor.js` on line `const dummyContainer = document.createDocumentFragment();`

in [fullcalendar-angular](https://github.com/fullcalendar/fullcalendar-angular) package.

Mine `package.json`:

```
    "@angular/animations": "^15.2.0",
    "@angular/common": "^15.2.0",
    "@angular/compiler": "^15.2.0",
    "@angular/core": "^15.2.0",
    "@angular/forms": "^15.2.0",
    "@angular/platform-browser": "^15.2.0",
    "@angular/platform-browser-dynamic": "^15.2.0",
    "@angular/router": "^15.2.0",
    "@fullcalendar/angular": "^6.1.7",
    "@fullcalendar/core": "^6.1.7",
    "@fullcalendar/daygrid": "^6.1.7",
```

**Solution:**

Inject `Document` via Angular `@Inject`, see my [PR](https://github.com/fullcalendar/fullcalendar-angular/pull/432).

**Test Case is not provided.**

**Comment by stehag** [comment]
This one is really important 👍🙏

**Comment by arshaw** [comment]
This is fixed in [v6.1.9](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.9)

Could someone please verify?

**Comment by stehag** [comment]
> This is fixed in [v6.1.9](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.9)
> 
> Could someone please verify?

Works like a charm!
